modules/alloy/src/main/ts/ephox/alloy/positioning/view/Bounder.ts

```diff
@@ -46,12 +46,16 @@ const calcReposition = (newX: number, newY: number, width: number, height: numbe
   const deltaW = Math.abs(Math.min(width, xInBounds ? boundsX + boundsWidth - newX : boundsX - (newX + width)));
   const deltaH = Math.abs(Math.min(height, yInBounds ? boundsY + boundsHeight - newY : boundsY - (newY + height)));
 
+  // measure the maximum x and y, taking into account the height and width of the element
+  const maxX = Math.max(bounds.x(), bounds.right() - width);
+  const maxY = Math.max(bounds.y(), bounds.bottom() - height);
+
   // TBIO-3366 + TBIO-4236:
   // Futz with the X position to ensure that x is positive, but not off the right side of the screen.
```

Comment:
Perhaps this change will require updating this comment.

Reply:
Still seems to be appropriate?

Reply:
I probably more meant it _looks_ like the behaviour has changed from preventing it going completely off the screen to preventing it from going partially off the screen. Not important, though. What someone reads from the comment is the most important thing about it.